airbyte-integrations/connectors/source-slack/source_slack/components.py

```diff
@@ -0,0 +1,156 @@
+from dataclasses import dataclass
+from typing import Optional
+from typing import List, Mapping, Any, Iterable
+import requests
+
+from airbyte_cdk.sources.declarative.auth.declarative_authenticator import DeclarativeAuthenticator
+from airbyte_cdk.sources.declarative.auth.token import BearerAuthenticator
+from airbyte_cdk.models import AirbyteMessage, SyncMode, Type, FailureType
+from airbyte_cdk.sources.declarative.types import Config, Record, StreamSlice, StreamState
+from airbyte_cdk.sources.declarative.extractors import DpathExtractor
+from airbyte_cdk.sources.declarative.transformations import RecordTransformation, AddFields
+from airbyte_cdk.sources.declarative.partition_routers import SubstreamPartitionRouter
+from airbyte_cdk.utils.traced_exception import AirbyteTracedException
+
+@dataclass
+class SlackAuthenticator(DeclarativeAuthenticator):
```

Comment:
can we use a selective authenticator instead of a custom one?

Reply:
moved to manifest with selective auth